servers/rendering/renderer_rd/shaders/cube_to_dp.glsl

```diff
@@ -35,22 +34,24 @@ layout(set = 0, binding = 0) uniform samplerCube source_cube;
 layout(push_constant, binding = 1, std430) uniform Params {
 	float z_far;
 	float z_near;
-	bool z_flip;
-	uint pad;
+	vec2 texel_size;
 	vec4 screen_rect;
 }
 params;
 
 void main() {
 	vec2 uv = uv_interp;
+	vec2 texel_size = abs(params.texel_size);
 
-	vec3 normal = vec3(uv * 2.0 - 1.0, 0.0);
+	const float padding = 1.0;
```

Comment:
This doesn't really do anything. Is there every a situation where it would be something other than 1.0?

Reply:
It was a left over from when I was testing various padding values, fixed.